templates/vpc-management.template

```diff
@@ -202,10 +202,7 @@ Parameters:
       orward slash (/).
     Type: String
 Conditions:
-  cGovCloudCondition:
-    !Equals
-    - !Ref AWS::Region
-    - us-gov-west-1
+  cGovCloudCondition: !Or [!Equals [!Ref "AWS::Region", us-gov-west-1], !Equals [!Ref "AWS::Region", us-gov-east-1]]
```

Comment:
```suggestion
  cGovCloudCondition: !Equals [!Ref "AWS::Partition", aws-us-gov]
```

recommend using the [`AWS::Partition` pseudoparameter](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html#cfn-pseudo-param-partition) instead